workflows/wf_titan_illumina_pe.wdl

```diff
@@ -157,7 +157,8 @@ workflow titan_illumina_pe {
     String  nextclade_aa_dels           = nextclade_one_sample.nextclade_aa_dels
     String  nextclade_version           = nextclade_one_sample.nextclade_version
 
-    File    ivar_tsv                    = variant_call.sample_variants
+    File    ivar_tsv                    = variant_call.sample_variants_tsv
+    File    ivar_vcf                    = variant_call.sample_variants_vcf
     String  ivar_variant_version        = variant_call.ivar_version
```

Comment:
@rpetit3 any thoughts on grabbing the variant_num at the workflow level as well?

Reply:
I guess we never grabbed in the pass. Disregard; I think there was some internal discussions on this already and there was a feeling that this wouldn't be particularly useful for PHLs, but may be informative for deeper investigations during which a dig into the intermediate files could be done